spacy/lang/es/tokenizer_exceptions.py

```diff
@@ -4,22 +4,33 @@
 from ...symbols import ORTH, LEMMA, NORM, PRON_LEMMA
 
 
-_exc = {
+# Not used by default as they are only used in certains regional speech modes, left here for completeness
+_slang_exc = {
```

Comment:
I don't think we want additional lists of exceptions. I feel like this may blow up and we might end up maintaining various incompatible lists, and it could be slightly subjective what exactly should go in which list. We should have reasonable rules for the "proper"/official language constructs, and if people want to model slang or other traits of specific language, they can write their own exceptions.

Reply:
I agree, I did it that way as I didn't know how you would want to handle the situation.
Should I modify the pull request, or is something you'll see from your side?